google/api_core/extended_operation.py

```diff
@@ -68,9 +71,14 @@ class ExtendedOperation(polling.PollingFuture):
     """
 
     def __init__(
-        self, extended_operation, refresh, cancel, retry=polling.DEFAULT_RETRY
```

Comment:
Right now, setting retry will be no-op. It will be a confusing user behavior. Ideally, we would warn developer somehow. I don't think we handle this in GAPIC right now. Something to consider - https://docs.python.org/3/library/exceptions.html#DeprecationWarning.

Reply:
Setting retry will not be no-op, it will overwrite settng polling for backward compatibility int `PollingFuture.__init__()`:
```
self._polling = kwargs.get("retry", polling)
```

The deprecation warning is not thrown to avoid contaminating users logs, since it will be affecting every current user.